app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  include Pagy::Frontend
+  
+  def formatted_date(datetime)
+    datetime.to_date.strftime("%d %b, %Y")
```

Comment:
what if datetime argument is nil or empty string?
We should always take care of such cases.

Reply:
set presence true in model for this.

Reply:
I have added `presence: true` in model for this so this should not be a problem moving forward.